src/utils/sidebar.js

```diff
@@ -142,21 +135,36 @@ function normalizeSidebar({
   return currentResult
 }
 
+function findChildWithSource(item) {
+  return item.source ? item : findChildWithSource(item.children[0])
+}
+
+/*
+ * Exports
+ */
+
+// Runs at module load time
 const normalizedSidebar = normalizeSidebar({
   data: sidebar,
   parentPath: ''
 })
 
-// Exports
-
+/**
+ * Finds `path` in sidebar struct
```

Comment:
again, not sure, how valuable this is ... this comment does not give more information to my mind and function itself is more or less self-descritptive

Reply:
Hey, sorry. This is still draft. Honestly I didn't expect you to review this until it became a real PR. Please wait, I will request your review soon 🙂